screens/OnboardingScreen/index.tsx

```diff
@@ -1,15 +1,35 @@
-import React, { useState } from "react";
+import React, { ReactElement, useState } from "react";
 import { StyleSheet } from "react-native";
-import Intro1 from "./Intro1";
-import Intro2 from "./Intro2";
-import Intro3 from "./Intro3";
-import Intro4 from "./Intro4";
-import Intro5 from "./Intro5";
+import Intro1 from "./Day1/Intro1";
+import Intro2 from "./Day1/Intro2";
+import Intro3 from "./Day1/Intro3";
+import Intro4 from "./Day1/Intro4";
+import Intro5 from "./Day1/Intro5";
+import Intro1Day2 from "./Day2+3/Intro1Day2";
+import Intro2Day2 from "./Day2+3/Intro2Day2";
+import Intro3Day2 from "./Day2+3/Intro3Day2";
 import { View } from "../../components/Themed";
 import Swiper from "react-native-swiper";
 import Colors from "../../constants/Colors";
 
-export default function OnboardingScreen() {
+interface OnboardingScreenProps {
+  views: ReactElement[];
+}
+
+export const Day1Screens = [
+  <Intro1 key={2235} />,
+  <Intro2 key={22342} />,
+  <Intro3 key={23262} />,
+  <Intro4 key={7622} />,
+  <Intro5 key={87922} />,
```

Comment:
Are these just random numbers or are they significant

Reply:
Super random. We just felt like 1-5 are soooooooooo boring.